packs/equipment-effects/effect-clawed-bracers.json

```diff
@@ -0,0 +1,62 @@
+{
+    "_id": "lh2YM9yNJixOC8RI",
+    "img": "icons/equipment/hand/gauntlet-clawed-steel.webp",
+    "name": "Effect: Clawed Bracers",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Clawed Bracers]</p>\n<p>You gain a climb Speed of 20 feet and a claw unarmed attack with the agile and finesse traits that deals 1d6 slashing damage.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 10
+        },
+        "level": {
+            "value": 7
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder GM Core"
+        },
+        "rules": [
+            {
+                "key": "BaseSpeed",
+                "selector": "climb",
+                "value": 20
+            },
+            {
+                "category": "unarmed",
+                "damage": {
+                    "base": {
+                        "damageType": "slashing",
+                        "dice": 1,
+                        "die": "d6"
+                    }
+                },
+                "group": "brawling",
```

Comment:
It doesn't specifically state brawling, but most claw attacks seem to be in that group?

Reply:
Some strikes don't have a group.